src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIExtension.groovy

```diff
@@ -33,6 +33,10 @@ class WPIExtension {
     String toolchainVersionLow = "6.3"
     String toolchainVersionHigh = "6.3"
 
+    // Set to true to use debug JNI
+    // Might require extra libraries (especially on windows)
+    boolean debugSimJNI = false
```

Comment:
Curiosity, why aren't you using the proper javadoc format for these comments? The Gradle Plugin publish plugin automatically generates a javadocs jar and publishes it. These won't be included if you don't use that format.

Reply:
What can see those docs? Especially when in the groovy DSL these are not super well typed.

Even so, we should do everything at one, so that shouldn't hold this PR.